pyfstat/core.py

```diff
@@ -1099,27 +1101,19 @@ def plot_twoF_cumulative(
         """
         if ax is None:
             fig, ax = plt.subplots()
-        if injectSources:
-            pfs_input = dict(
-                h0=injectSources["h0"],
-                cosi=injectSources["cosi"],
-                psi=injectSources["psi"],
-                Alpha=injectSources["Alpha"],
-                Delta=injectSources["Delta"],
-                Freq=injectSources["fkdot"][0],
-            )
-        else:
-            pfs_input = None
+            ax.grid()
 
-        taus, twoFs = self.calculate_twoF_cumulative(**kwargs)
+        taus, twoFs = self.calculate_twoF_cumulative(**calculate_twoF_cumulative_kwargs)
         ax.plot(taus / 86400.0, twoFs, label=plt_label, color=c)
         if len(self.detector_names) > 1:
             detector_names = self.detector_names
             detectors = self.detectors
             for d in self.detector_names:
                 self.detectors = d
                 self.init_computefstatistic()
-                taus, twoFs = self.calculate_twoF_cumulative(**kwargs)
+                taus, twoFs = self.calculate_twoF_cumulative(
+                    **calculate_twoF_cumulative_kwargs
+                )
                 ax.plot(
                     taus / 86400.0,
                     twoFs,
```

Comment:
```suggestion
                taus, twoFXs = self.calculate_twoF_cumulative(
                    **calculate_twoF_cumulative_kwargs
                )
                ax.plot(
                    taus / 86400.0,
                    twoFXs,
```
Otherwise, at the end it returns these per-IFO values instead of the arguably expected multi-IFO ones.